Makefile

```diff
@@ -27,6 +27,7 @@ WIN_BUILD_OVS_TAG  := $(NANOSERVER_VERSION)-$(WIN_OVS_VERSION)
 GIT_HOOKS          := $(shell find hack/git_client_side_hooks -type f -print)
 DOCKER_NETWORK     ?= default
 TRIVY_TARGET_IMAGE ?=
+GOCOVERDIR ?= "/tmp/coverage"
```

Comment:
why do we need to define this in the Makefile? I thought this variable was only consumed at runtime, not when building instrumented binaries.

Reply:
Ah, first I was trying to create the directory in the Makefile. Must have slipped by me to remove it😅